### PR TITLE
make the proxies order the same as the order in config file

### PR DESCRIPTION
**Pull request by ezksd**
I had create a pull request to clash to fix the proxies arbitrary order problem, some code in the clash-dashboard still need to fix. i push to your repository directly few minutes ago, it's a mistake.

### CODE REVIEW

File: src/components/Tags/index.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import classnames from 'classnames'
 import './style.scss'
 
 interface TagsProps extends BaseComponentProps, I18nProps {
-    data: Set<string>
+    data: [string]
```
Comment (Dreamacro) [inline src/components/Tags/index.tsx]: should be `string[]`, `[string]` means an array only have an element.

File: src/containers/Proxies/components/Group/index.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ export class Group extends React.Component<GroupProps, {}> {
 
     render () {
         const { config } = this.props
-        const proxies = new Set(config.all)
+        // const proxies = new Set(config.all)
```
Comment (Dreamacro) [inline src/containers/Proxies/components/Group/index.tsx]: should remove this line

**Comment by ezksd** [comment]
sorry for that, I'm new to typescript

**Comment by Dreamacro** [comment]
@ezksd no problem, just have a try

**Pull request merged**